subt/artf_node.py

```diff
@@ -84,13 +84,13 @@ def __init__(self, config, bus):
         self.fx = config.get('fx', 554.25469)  # use drone X4 for testing (TODO update all configs!)
 
     def create_detector(self):
-        model = os.path.join(os.path.dirname(__file__), '../../../mdnet1.64.64.13.4.relu.pth')
+        model = os.path.join(os.path.dirname(__file__), '../../../mdnet3.64.64.13.4.relu.pth')
         confidence_thresholds = {
             'survivor': 0.8,
-            'backpack': 0.95,
-            'phone': 0.77,
-            'helmet': 0.9,
-            'rope': 0.8
+            'backpack': 0.8,
+            'phone': 0.5,
```

Comment:
Together with the 0.1 threshold for the other detector, you are basically relying on the fact that if both of the detectors weakly suspect there is a phone there, it really is?

Reply:
>if both of the detectors weakly suspect there is a phone there, it really is

Yes, I did not find any false detection. It seems that already the combination of the detector is sufficiently robust. It seems to me, that the risk is acceptable in case phone.

However, I see your concern and because there are just one point near to 0.5 threshold I am willing to shift the value between 0.7 and 0.8. Again, I would like to note that it is only sample of the true detection.